src/cli/serve/serve.js

```diff
@@ -169,13 +168,6 @@ export default function (program) {
       '-l, --log-file <path>',
       'Deprecated, set logging file destination in your configuration'
     )
-    .option(
-      '--plugin-dir <path>',
-      'A path to scan for plugins, this can be specified multiple ' +
```

Comment:
I checked, and neither this cli option or the `plugins.scanDirs` config prop seems to be documented anywhere in our public (or internal) docs.

Reply:
I think we can also remove the `--plugins` option below then (and `pluginDirCollector`)? As it appears to just be an alias for `--plugin-dir`

Reply:
(I guess that also means we would need to deprecate it in 7.x too?)

Reply:
No, `--plugins` allow to specify paths to specific plugins (one path per plugin, no folder scan), and got mapped to `plugins.paths` instead of `plugins.scanDirs`, which is still supported and used in 8

Reply:
and yea, will need to open a follow up of https://github.com/elastic/kibana/pull/113366 to deprecates it in 7.x